lib/stellar/client.rb

```diff
@@ -54,6 +56,9 @@ def initialize(options)
           "X-Client-Version" => VERSION,
         }
       end
+      # TODO: add a network_passphrase parameter per client call like other SDK's 
+      # instead of setting a default for the module.
+      Stellar.default_network = options[:network]
```

Comment:
This line is going to have side-effects on anyone using two clients. That's probably and unlikely case, but it would be shocking if in a next release someone had instantiated two clients one for test and one for public, and the default_network value had been altered without them knowing.

So I think this is a breaking change and we shouldn't make it. Is there a lot of value in doing this ahead of doing the TODO? The TODO outlines where the real big wins are.

Reply:
I guess it doesn't really add much value, I can revert the commit.